### DISCUSSION TITLE
Requesting User does not have permission to update. When submitting new Catalog Item via PAC CLI

### DISCUSSION THREAD

**Discussion by mrmwinkjust** [microsoft/PowerPlatform-Catalog#20] [Q&A] [answered]
When attempting to submit a new catalog item it always has a status of "Failed Prevalidation", upon investigation the full error is "Requesting User does not have permission to update".

I have confirmed I am SA, I've also added all the Catalog Security Roles to confirm I have the required access but still get the same.

1. Create simple Catalog submission document (Attached)
2. In Pac CLI (v1.30.6) run command - pac catalog submit -env "{env}" -p "{path to config}" -sz "{solution.zip}"
3. Check tracking for result

Added screenshot of note attached to Failed Validation item in Catalog App

- Browser EDGE

[submission-example.json](https://github.com/microsoft/PowerPlatform-Catalog/files/14164393/submission-example.json)
![CatalogError](https://github.com/microsoft/PowerPlatform-Catalog/assets/143486631/21cbffa9-9101-4163-a7d0-6b7261f4f6eb)

**Comment by MattB-msft** [comment]
That error means that the user of the Pac command is not a member of the publisher you're trying to push an update to. 
To push an update, the requesting user must be a member of that publisher. 

To do that you would add the user to the Publishers User list in catalog, under the publisher. 
You can also do this as part of a submit by sending a group of users or AAD groups during the submission request.

**Comment by mrmwinkjust** [comment, marked as answer]
Awesome, that solved it thanks, just getting to grips with the Catalog and seeing how it may integrate in the future.
